src/elements/OcButton.vue

```diff
@@ -4,15 +4,14 @@
     :href="href"
     :to="to"
     :type="submit"
-    :text="text"
     :aria-label="ariaLabel"
     :class="$_ocButton_buttonClass"
     @click="onClick"
   >
-    <oc-icon v-if="icon" :class="$_ocButton_iconClass" :name="icon"></oc-icon>
-    <slot>
-      <span v-if="text" v-text="text" />
-    </slot>
+    <oc-icon v-if="icon" :class="$_ocButton_iconClass" :name="icon" />
+    <span>
```

Comment:
This `<span>` will also show up in icon-only buttons and add an unwanted space on the right.
Let's `v-if` this.

Reply:
Done